install.sh

```diff
@@ -108,26 +147,26 @@ function install() {
         # ensure folders exists
         for SUBCONFIG_FOLDER in $SUBCONFIG_FOLDERS ; do
             SUBCONFIG_FOLDER=$(sanitizePath "$SUBCONFIG_FOLDER")
-            echo "creating '/usr/lib/systemd/$SERVICE/$SUBCONFIG_FOLDER'"
-            (cd "/usr/lib/systemd/" && cd "$SERVICE" && sudo mkdir -p "$SUBCONFIG_FOLDER")
+            echo "creating '$PREFIX_DIR/lib/systemd/$SERVICE/$SUBCONFIG_FOLDER'"
+            (cd "$PREFIX_DIR/lib/systemd/" && cd "$SERVICE" && mkdir -p "$SUBCONFIG_FOLDER")
         done
```

Comment:
1. The script fails if `lib/systemd/$SERVICE` (resolves to `lib/systemd/system`) does not exist as a directory yet (as is the case for AUR builds from scratch) as it will first try to create `fw-fanctrl.service` but won't create its directory.
2. The script will fail next if `lib/systemd/$SERVICE/$SUBCONFIG_FOLDER` (`lib/systemd/system/system-sleep`) does not exist yet in the subconfigs loop right afterwards. The script has `mkdir -p` calls for the subconfig - but this one only runs if the subconfig provides *sub*directories of its own which is currently never the case.